modules/local/openms_featurefinderidentification.nf

```diff
@@ -2,13 +2,13 @@ process OPENMS_FEATUREFINDERIDENTIFICATION  {
     tag "$meta.id"
     label 'process_medium'
 
-    conda "bioconda::openms=2.9.1"
+    conda "bioconda::openms=3.0.0"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/openms:2.9.1--h135471a_1' :
-        'biocontainers/openms:2.9.1--h135471a_1' }"
+        'https://depot.galaxyproject.org/singularity/openms:3.0.0--h8964181_1' :
+        'biocontainers/openms:3.0.0--h8964181_1' }"
 
     input:
-    tuple val(meta), path(id_quant_int), path(mzml), path(id_quant)
+    tuple val(meta), path(mzml), path(id_int), path(id_ext)
```

Comment:
Did you change the order in the mchquant workflow?

Reply:
Yes I did. I thought it is more intuitive since the FFI input can be mzml, idxml (internal), idxml(external) or mzml, idxml(external)

Reply:
Perfect then this is fine